specification/azsadmin/resource-manager/containerregistry/Microsoft.ContainerRegistry.Admin/preview/2019-11-01/configuration.json

```diff
@@ -0,0 +1,264 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-11-01",
+    "title": "ContainerRegistryManagementClient",
+    "description": "Admin Container Registry Management Client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry.Admin/locations/{location}/configuration/{configurationName}": {
+      "put": {
+        "x-ms-examples": {
+          "Returns container registry overall configuration properties.": {
+            "$ref": "./examples/Configuration/PutConfiguration.json"
+          }
+        },
+        "tags": [
+          "ContainerRegistryConfiguration"
+        ],
+        "description": "Configure container registry overall configuration properties.",
+        "operationId": "ContainerRegistryConfiguration_Put",
```

Comment:
same comment on using ContainerRegistry as the prefix for everything.

Reply:
It is ok, we don't want to collide with Storage Api for example which also has similar operations.

Reply:
It wont collide as the prefix will be ContainerRegistry_ConfigurationCreateOrUpdate